TwitchDownloaderCore/TwitchObjects/ChatRoot.cs

```diff
@@ -212,6 +212,7 @@ public class EmbedChatBadge
     {
         public string name { get; set; }
         public Dictionary<string, byte[]> versions { get; set; }
+        public Dictionary<string, string> urls { get; set; }
```

Comment:
Add a `JsonIgnore` attribute so this isn't written to chat JSONs.
```suggestion
        [SystemText::JsonIgnore(Condition = SystemText.JsonIgnoreCondition.WhenWritingNull)]
        public Dictionary<string, string> urls { get; set; }
```